recipients_app/lib/view/widgets/survey/survey_card_bottom_action.dart

```diff
@@ -59,27 +64,30 @@ class SurveyCardBottomAction extends StatelessWidget {
     );
   }
 
-  void _navigateToSurvey(BuildContext context) {
+  void _navigateToSurvey(BuildContext context, SurveyCubit surveyCubit) {
```

Comment:
This is the place where you could just use the context.read to get the surveyCubit and pass it on. Remember: if it's on top of the MaterialApp you would not need to do that - its automatically available in the surveypage

Reply:
I couldn't. For some reason the buildContext in that place didn't have SurveyCubit, it throws errors for me. It seems that when the card is created there is buildContext different then later on 🤷

Reply:
fixed it